git_machete/cmd.py

```diff
@@ -444,40 +460,40 @@ def down(b, pick_mode):
         return "\n".join(dbs)
 
 
-def first_branch(b):
+def first_branch(b: str) -> Optional[str]:
```

Comment:
Hmmm why optional? I can't see an execution path that would lead to `None` getting returned 🤔

Reply:
good point